plugins/org.locationtech.udig.project/src/org/locationtech/udig/project/internal/render/impl/ViewportModelImpl.java

```diff
@@ -685,7 +685,7 @@ public void setBounds(double minx, double maxx, double miny, double maxy) {
      */
     public AffineTransform worldToScreenTransform() {
         if (!validState())
-            return null;
+            return new AffineTransform(); //Identity (screen-space is arbitrary here.)
```

Comment:
Aha, now I got the puzzle and it looks like this change addresses requested Null-check in pull-request #341 ([review discussion](https://github.com/locationtech/udig-platform/pull/341#discussion_r278718919))

Reply:
It would be great to have another test for worldToScreenTransform for both cases, validateState is true or false ;)